jenkins-agents/ansible/roles/common/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
   yum:
     pkg: ['wget', 'mc', 'vim', 'unzip', 'iputils', 'bind-utils', 'findutils',
           'htop', 'screen', 'lsof', 'bash-completion', 'bzip2', 'rng-tools',
-          'ansible', 'zip', 'python2-pip', 'jq', 'libXScrnSaver', 'python3', 'python3-devel',
```

Comment:
Just curious, why is this being removed, is there a conflict?

Reply:
it shouldn't be removed. My fault